packages/eas-cli/src/build/local.ts

```diff
@@ -6,7 +6,7 @@ import semver from 'semver';
 import { ora } from '../ora';
 
 const PLUGIN_PACKAGE_NAME = 'eas-cli-local-build-plugin';
-const PLUGIN_PACKAGE_VERSION = '0.0.58';
+const PLUGIN_PACKAGE_VERSION = '0.0.62';
```

Comment:
- FWIW this is bumped automatically before every release. See https://github.com/expo/eas-cli/blob/main/scripts/src/updateLocalPlugin.sh The version is resolved based on the `latest` tag of the package.
- I think the changes you need are already in version `0.0.61` which is `latest`.
- If we don't bump the `latest` tag, the version you set here will be changed anyway.

Reply:
I'd recommend changing this to `0.0.61` or leaving as it is.

Reply:
I bumped it manually, because I couldn't test this functionality with 0.0.58, so I had to bump it manually. I ran into that env-paths issues with 0.0.61, that's why I set it to 0.0.62, so I can confirm the fix as soon as that releases.

Reply:
For testing you can use 
`EAS_LOCAL_BUILD_PLUGIN_PATH=/path-to-eas-build-repo/bin/eas-cli-local-build-plugin eas build --local`